dataplex/snippets/src/main/java/dataplex/UpdateEntryGroup.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dataplex;
+
+// [START dataplex_update_entry_group]
+import com.google.cloud.dataplex.v1.CatalogServiceClient;
+import com.google.cloud.dataplex.v1.EntryGroup;
+import com.google.cloud.dataplex.v1.EntryGroupName;
+import com.google.protobuf.FieldMask;
+
+// Sample to update Entry Group
+public class UpdateEntryGroup {
+
+  public static void main(String[] args) throws Exception {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "MY_PROJECT_ID";
+    // Available locations: https://cloud.google.com/dataplex/docs/locations
+    String location = "MY_LOCATION";
+    String entryGroupId = "MY_ENTRY_GROUP_ID";
+
+    EntryGroupName entryGroupName = EntryGroupName.of(projectId, location, entryGroupId);
+    EntryGroup updatedEntryGroup = updateEntryGroup(entryGroupName);
+    System.out.println("Successfully updated entry group: " + updatedEntryGroup.getName());
+  }
+
+  public static EntryGroup updateEntryGroup(EntryGroupName entryGroupName) throws Exception {
+    EntryGroup entryGroup =
+        EntryGroup.newBuilder()
+            .setName(entryGroupName.toString())
```

Comment:
if the only usage of the `entryGroupName` is to use its fully qualified name, why not to pass that name to the method?